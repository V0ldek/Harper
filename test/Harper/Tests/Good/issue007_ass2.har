fun :: Integer -> Integer;
fun n = {
  var (y :: Integer) = n;
  var (n :: Integer) = n;
  n *= 2;
  return n;
};

main :: Integer;
main = fun 42;