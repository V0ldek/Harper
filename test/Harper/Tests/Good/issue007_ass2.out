Execution ended with value: 84